docs/source/conf.py

```diff
@@ -16,7 +16,6 @@
 author = 'The PHP Group'
 extensions = [
     'sphinx_design',
-    'sphinxawesome_theme.highlighting',
```

Comment:
Why is this dropped?

Reply:
Deprecated, details are in the commit message @iluuu1994